Clean up formatting and added small functionality

KeriBrady opened the pull request:
Create mulitple conditions for while loop - to prevent a correct guess from cycling though more prompts

Add hints to prompt for incorrect guesses (too high or too low).

Commit:
Clean up formatting and added small functionality

Create mulitple conditions for while loop - to prevent a correct guess from cycling though more prompts

Add hints to prompt for incorrect guesses (too high or too low).

mel-quark commented on jan5_assign.html:
```diff
@@ -1,32 +1,37 @@
-  while(count < 3){
+  pass = false;
+  
+  while((count < 3) && (pass == false)) {
```
because pass has a boolean value, you don't have to compare it to false.
`pass == false` is equivalent to `!pass`
so this line could be 
`while((count < 3) && !pass) {`

mel-quark commented on jan5_assign.html:
```diff
@@ -1,32 +1,37 @@
     else {
-      guess = prompt("Try again!");
+      guess = prompt("Too low, try again!");
       count++;
```
`count++` should happen no matter the result of the `if`, so you can have it only once (outside of the if).

Commit:
3 small formatting changes

Line 32, change to '!pass' from 'pass == false' for same reason
Line 29, move two instances of 'count++' to outside if loop statement
